src/applications/disability-benefits/all-claims/pages/privateMedicalRecordsRelease.js

```diff
@@ -94,7 +97,27 @@ export const uiSchema = {
 export const schema = {
   type: 'object',
   properties: {
-    providerFacility: providerFacilities,
+    providerFacility: {
```

Comment:
Why is this defined explicitly in here now?

Reply:
Hey @cvalarida! I had to switch it because the schema has the treatmentDateRange as an array, here we use it as an object. The reason for that is...UI should not be accepting multiple treatmentDateRange's (yet), but the API & schema should be set up to do so since the form allows for it.